components/Navbar/type.ts

```diff
@@ -0,0 +1,13 @@
+export type Resource = {
```

Comment:
These types are unnecessary, the types are inferred by the array literal hardcoded configs.